### PR TITLE
Fixes tab cycling when a revealable input exists in a form

### LINKED ISSUE #1583: Pressing Tab during new account creation does not properly cycle through input fields
## Problem Definition

When making a new account users who are used to cycling through input fields by pressing "Tab" will have a problem as it does not work. Check the gif :point_down: 
Peek 2020-10-09 23-34

I assume it's due to the "reveal password" button also getting selected during tab cycling.

## Task

Fix it ... probably by somehow ignoring the "Reveal password" button during the tab cycling

### PR DESCRIPTION
**Pull request by kelsos**
Closes #1583

### CODE REVIEW

File: frontend/app/src/components/inputs/RevealableInput.vue [hunk 1]
```diff
@@ -14,8 +13,13 @@
     :error-messages="errorMessages"
     v-on="$listeners"
     @input="input"
-    @click:append="revealed = !revealed"
-  />
+  >
```
Comment (LefterisJP) [inline frontend/app/src/components/inputs/RevealableInput.vue:16]: Proably this needs a `/`?

**Review by LefterisJP — APPROVED** [review]
LGTM but a unit test is failing in the frontend. Giving it a tentative approval as it worked when I tested it.

File: frontend/app/src/components/inputs/RevealableInput.vue (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/components/inputs/RevealableInput.vue:16]: No, now it has the template inside so it is not self-closing. it has a closing tag

**Comment by kelsos** [comment]
OK makes sense the test was testing with the appended icon, but now that changed since I could not change the tabIndex to that. Easy to fix.

**Pull request merged**